tests/data_files/Makefile

```diff
@@ -1558,6 +1558,77 @@ pkcs7_data_cert_signeddata_sha256.der: pkcs7_data_cert_signed_sha256.der
 	dd if=pkcs7_data_cert_signed_sha256.der of=$@ skip=19 bs=1
 all_final += pkcs7_data_cert_signeddata_sha256.der
 
+################################################################
+#### Generate C format test certs header
+################################################################
+
+# FIXME: Makefile will update those input files which is not expected.
+#        Remove them from dependancy to workaround it.
+#        For timebeing, we will always update it when `make ../src/test_certs.h`
```

Comment:
Do we still need this workaround? And please be aware that string `PolarSSLTest` couldn't be a dependancy.